### PR TITLE
fix: SwiftLint warnings in`Notification Engine`, `Protos`, `Request Strategy`, `Sync Engine`

### PR DESCRIPTION
**Pull request by agisilaos**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

With this PR we fix SwiftLint warnings in the following frameworks:

- Notification Engine
- Protos
- Request Strategy
- Sync Engine

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by netbe — APPROVED** [review]

File: wire-ios-protos/Protos/messages.pb.swift
Comment (David-Henner) [inline wire-ios-protos/Protos/messages.pb.swift:1]: fyi this file is auto generated and the SwiftLint fixes will be overridden with the next update

File: wire-ios-protos/Protos/mls.pb.swift
Comment (David-Henner) [inline wire-ios-protos/Protos/mls.pb.swift:1]: same here

File: wire-ios-protos/Protos/otr.pb.swift
Comment (David-Henner) [inline wire-ios-protos/Protos/otr.pb.swift:1]: and here

**Review by David-Henner — APPROVED** [review]

File: wire-ios-protos/Protos/messages.pb.swift
Comment (agisilaos) [reply wire-ios-protos/Protos/messages.pb.swift:1]: I will disable SwiftLint for that file

File: wire-ios-protos/Protos/mls.pb.swift
Comment (agisilaos) [reply wire-ios-protos/Protos/mls.pb.swift:1]: I will disable SwiftLint for that file

File: wire-ios-protos/Protos/otr.pb.swift
Comment (agisilaos) [reply wire-ios-protos/Protos/otr.pb.swift:1]: I will disable SwiftLint for that file

**Pull request merged**